README.md

```diff
@@ -23,6 +26,8 @@ console.log(results);
 
 ```js
 // Your answer...
+Results would be a list of all documents that have the the name "Bob". AuthorModel is usually used when making a new model and creating a seed file.
+In this case you would want to use the .findOne() function.
```

Comment:
The issue is that a callback needs to be passed in as a second argument to `findOne`. In this callback we can `console.log` the "found" author.